tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh

```diff
@@ -100,7 +102,7 @@ function create_framework_archive {
   cp "LICENSE" ${TFLS_TMPDIR}
 
   # Unzip the iOS framework zip generated by bazel to TFLS_TMPDIR
-  unzip ${IOS_FRAMEWORK_PATH} -d "${TFLS_TMPDIR}"/Frameworks  
+  unzip $1 -d "${TFLS_TMPDIR}"/Frameworks  
```

Comment:
similar comment as above:

```sh
IOS_FRAMEWORK_PATH="$1"
unzip "${IOS_FRAMEWORK_PATH}" -d "${TFLS_TMPDIR}"/Frameworks
```

The double quotes around the variable here would be needed in case the framework path contains any spaces in it.

Reply:
Fixed